restaurants/src/app/components/restaurants/restaurants.component.html

```diff
@@ -28,4 +30,4 @@ <h2>{{restaurants[selectedRestaurant].name | uppercase}} Details</h2>
 
   <div *ngIf="selectedRestaurant">
     <button type="button" (click)="delete()">Delete Restaurant</button>
-  </div>
\ No newline at end of file
+  </div>
```

Comment:
can lines 21 through 33 be deleted?

Reply:
gonna make sure it works with it commented out first